features/step_definitions/factory_steps.rb

```diff
@@ -50,8 +50,10 @@
 
 Given /the following volunteer opportunities exist/ do |volunteer_ops_table|
   volunteer_ops_table.hashes.each do |volunteer_op|
-    volunteer_op["organisation"] = Organisation.find_by_name(volunteer_op["organisation"])
-    VolunteerOp.create! volunteer_op
+    VCR.use_cassette("#{volunteer_op['name']}-#{volunteer_op['postcode']}") do
```

Comment:
do we need this explicit VCR cassette setting? isn't the whole scenario wrapped in VCR?

Personally I lean towards just having every feature wrapped in VCR ...